pkg/virtctl/vm/vm.go

```diff
@@ -104,12 +105,33 @@ func NewMigrateCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	return cmd
 }
 
+func NewRenameCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "rename [vm_name] [new_vm_name]",
+		Short:   "Rename a stopped virtual machine.",
+		Example: usage(COMMAND_RENAME),
+		Args:    cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := Command{command: COMMAND_RENAME, clientConfig: clientConfig}
+			return c.Run(cmd, args)
+		},
+	}
+	cmd.SetUsageTemplate(templates.UsageTemplate())
+	return cmd
+}
+
 type Command struct {
 	clientConfig clientcmd.ClientConfig
 	command      string
 }
 
 func usage(cmd string) string {
+	if cmd == COMMAND_RENAME {
+		usage := fmt.Sprintf("	# %s a virtual machine called 'myvm' to 'notmyvm'\n", strings.Title(COMMAND_RENAME))
```

Comment:
Maybe just write `rename` here? If someone will change the command from `rename` to `rename-vm` the grammar here will be wrong...

Reply:
Fixed